lib/kramdown-rfc2629.rb

```diff
@@ -495,8 +501,10 @@ def convert_img(el, indent, opts) # misuse the tag!
           end
         end
         if alt == ":include:"   # Really bad misuse of tag...
+          label = el.attr.delete('label')
+          warn "*** no label for anchor #{anchor}" unless label
           to_insert = ""
-          anchor.scan(/(W3C|3GPP|[A-Z-]+)[.]?([A-Za-z0-9.-]+)/) do |t, n|
+          anchor.scan(/(W3C|3GPP|[A-Z-]+)[.]?([A-Za-z0-9.\/-]+)/) do |t, n|
```

Comment:
anchors are XML names, and / is not actually allowed in an XML name, so  DOI anchors will always need to have an _ for the /.  (This is not a problem if the DOI reference is used with a custom anchor, hmm.)